tests/test_dcim.py

```diff
@@ -401,6 +401,28 @@ def test_get_units(self, mock):
             isinstance(ret[0].device, pynetbox.models.dcim.Devices)
         )
 
+    @patch(
+        'pynetbox.core.query.requests.sessions.Session.get',
+        side_effect=[
+            Response(fixture='dcim/rack.json'),
+            Response(fixture='dcim/rack_u.json'),
```

Comment:
Is it really the same return as `units`?

Reply:
Yes just got this from netboxdemo.com

```

{
    "count": 12,
    "next": null,
    "previous": null,
    "results": [
        {
            "id": 12,
            "name": "U12",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": {
                "id": 16,
                "url": "https://netboxdemo.com/api/dcim/devices/16/",
                "name": "DS9_SECOFFICE_FIBER_U12",
                "display_name": "DS9_SECOFFICE_FIBER_U12"
            }
        },
        {
            "id": 11,
            "name": "U11",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": {
                "id": 20,
                "url": "https://netboxdemo.com/api/dcim/devices/20/",
                "name": "DS9_SECOFFICE_U11",
                "display_name": "DS9_SECOFFICE_U11"
            }
        },
        {
            "id": 10,
            "name": "U10",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": null
        },
        {
            "id": 9,
            "name": "U9",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": null
        },
        {
            "id": 8,
            "name": "U8",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": null
        },
        {
            "id": 7,
            "name": "U7",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": null
        },
        {
            "id": 6,
            "name": "U6",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": null
        },
        {
            "id": 5,
            "name": "U5",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": {
                "id": 26,
                "url": "https://netboxdemo.com/api/dcim/devices/26/",
                "name": "DS9_SECOFFICE_ACCESS",
                "display_name": "DS9_SECOFFICE_ACCESS"
            }
        },
        {
            "id": 4,
            "name": "U4",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": null
        },
        {
            "id": 3,
            "name": "U3",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": null
        },
        {
            "id": 2,
            "name": "U2",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": null
        },
        {
            "id": 1,
            "name": "U1",
            "face": {
                "value": "front",
                "label": "Front",
                "id": 0
            },
            "device": {
                "id": 18,
                "url": "https://netboxdemo.com/api/dcim/devices/18/",
                "name": "DS9_SECOFFICE_PDU",
                "display_name": "DS9_SECOFFICE_PDU"
            }
        }
    ]
}
```

Reply:
oh I see, didn't know units was replaced. This works then. Sorry for the confusion.